vtex-sites/base.store
# Fix: remove duplicate attribute that was causing the product picture to blink when added to cart

## What's the purpose of this pull request?
Fix the blinking product picture when it was added to cart

## How it works? 
It removes a property which was being updated shortly after, causing a re-rendering of the component.

## How to test it?
Add a product to cart. See that the picture does not blink. You can check it blinks when reverting to the previous commit, or just running in the master branch.
